python/nano/scripts/bigdl-nano-init

```diff
@@ -318,7 +349,7 @@ else
         echo "nano_vars.sh already exists"
     elif [ ! -f $CONDA_PREFIX/bin/bigdl-nano-init ]; then
         echo "It seems that you are using bidl-nano installed by system's pip, "
```

Comment:
another typo `bigl-nano`

and may change the `Cannot find bigdl-nano-init` in line 376 to `Cannot find bigdl-nano-unset-env`